src/Extensions/Compatibility.UlsLoggerAdapter/UlsLogAddapter.cs

```diff
@@ -0,0 +1,82 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using System;
+using System.Collections.Concurrent;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Logging;
+using Microsoft.Omex.System.Logging;
+
+namespace Microsoft.Omex.Extensions.Compatibility.UlsLoggerAdapter
+{
+	/// <summary>
+	/// Hosted service subscribes to log events from Microsoft.Omex.System.UlsLogger and propagates them to ILogger
+	/// </summary>
+	internal class UlsLogAddapter : IHostedService
```

Comment:
Why do we need a hosted service for logging? Isn't the logger implementation/registration enought?